test/python/CMakeLists.txt

```diff
@@ -223,6 +223,14 @@ teca_add_test(py_test_bayesian_ar_detect_mpi_threads
     FEATURES ${TECA_HAS_NETCDF} ${TECA_HAS_MPI}
     REQ_TECA_DATA)
 
+teca_add_test(py_test_deeplabv3p_ar_detect
+    COMMAND  ${PYTHON_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/test_deeplabv3p_ar_detect.py
+    "${TECA_DATA_ROOT}/pre_trained_deeplab_resnet_IVT_artmip_probabilistic_adam_epoch_001_loss_perceptual_loss_160987.8365_time_corr_split_state_dict_final.pt"
```

Comment:
This is the longest file name I have seen in something like 20 years of software development. could we use a shorter file name?

Reply:
Hahahahaha

Reply:
@taobrienlbl Do you have suggestions for the filename?

Reply:
Could be deeplab_IVT_model.pt?

Reply:
How about `cascade_deeplab_IVT.pt`?

Reply:
sounds good!